runtime/common/src/main/kotlin/kotlinx/serialization/Shorthands.kt

```diff
@@ -47,6 +52,16 @@ fun SerialDescriptor.getElementIndexOrThrow(name: String): Int {
     return i
 }
 
-@Deprecated("Obsolete name from preview version of library.", ReplaceWith("elementsCount"))
+/**
+ * Searches for annotation of type [A] in annotations, obtained via
+ * [SerialDescriptor.getElementAnnotations] at given [elementIndex]
+ *
+ * Returns null if there is 0 or more than 1 annotation with such type.
+ */
+inline fun <reified A: Annotation> SerialDescriptor.findAnnotation(elementIndex: Int): A? {
```

Comment:
Let's move it to samples?
Currently, it does not carry any value in runtime core

Reply:
I think it can be useful in some applications. I can try to identify where I can use this function in the runtime library itself.

Reply:
Let's make it internal for a while